cfgov/v1/models/learn_page.py

```diff
@@ -106,6 +107,47 @@ class DocumentDetailPage(AbstractFilterPage):
     template = 'document-detail/index.html'
 
 
+class StoryPage(AbstractFilterPage):
```

Comment:
There's a lot of stuff mixed into those files right now, but maybe for the scope of this PR it might be best to just make a new file for your new page.

It's probably fair to say that the philosophy about when to break out things into separate apps is still evolving (@willbarton @rosskarchner). The `jobmanager` app is separate and has its own page model and related things, but that's mostly because it has some Django models. It doesn't seem like this Story PR has enough to warrant being separate.